### ISSUE TITLE
Prototype 2 Playtest

### ISSUE THREAD

**Issue by sebchampoux** [alexasalois/cart315-2021#4]
> Is the cleaning up while procrastinating something relatable to you? If not, what other tasks do you do instead of what you’re supposed to do?

Unfortunately, yes.  When I'm procrastinating something I _really_ don't want to do, there are few things that I won't do in order to not complete the task I should.

> Do you think the entire house would get too boring to clean?

Not necessarily.  If all the "mess" is cleaned up in the same way, then yes it would get boring.  But if there are additional challenges related to the mess , or the furniture, or other people in the house, then it could stay engaging.  Your concept actually makes me think of Luigi's Mansion (one of my favorite games ever).  The goal is always more or less the same throughout the mansion, capture ghosts and move from room to room, but most ghosts can only be caught by performing a specific sequence of actions.  The puzzle-ish nature of the game keeps it new .

> Do you have any suggestions for “surprise” events to pop up throughout the game? (maybe an online class that you need to run back to the bedroom to move your move / type in the chat, and you only have 30 seconds or you’ll get ‘caught’)

That's a good idea.  Here are two others that I thought of:
* Challenges related to the furniture.  For example, you need to get some item under your bed, or the sofa, and each piece of furniture must be displaced in a specific way.
* Challenges related to other inhabitants of the house.  Say, you want to get your phone in the dining room, but your mom is in the living room next door, and she'll be very frustrated if she sees you not working as your last semester's grades were terrible.

> When you think about this concept, what kind of “feel” are you getting? Is it a stressful / time based game? Or is it a chill / relaxing game where at the end you just feel satisfied of everything you cleaned up?

When I was reading your design journal, and looking at the atmosphere of the room you designed, it gave me the impression of a stressful game.  Often, when you procrastinate, time is of the essence.  So, it would emulate in the game the feeling you get when you procrastinate something just a lil' too much, and then you have no choice but to pull an anxiety-ridden all-nighter to study for your exam :( .

TLDR, even though there wasn't too much to test, the concept has potential in my opinion and I'd like to see more!

**Comment by PatrickChavez** [comment]
**1. Is the cleaning up while procrastinating something relatable to you? If not, what other tasks do you do instead of what you’re supposed to do?**
Cleaning up while procrastinating is something I do occasionally. Other activities include browsing the internet or playing video games. 
 
**2. Do you think the entire house would get too boring to clean?**
It would definitely be ambitious as a concept. Perhaps limiting the scope to a few rooms that are packed with stuff to clean (both hidden and in pain sight) could be an alternative? As long as the rooms/areas have interesting elements to them (maybe gameplay-based gimmicks and unique visuals), they should be able to keep the player entertained.
 
**3. Do you have any suggestions for “surprise” events to pop up throughout the game? (maybe an online class that you need to run back to the bedroom to move your move / type in the chat, and you only have 30 seconds or you’ll get ‘caught’)?**

- There could be a system where your parents are wandering around the house and them spotting or touching you could result in a game over.
- A “dust” mechanic where some objects would quickly become dusty over time, requiring the player to clean them up again.
- The room suddenly turning dark, forcing the player to stumble around the area until they can find a light source.
- Having a room that consists of nothing but doors: the player has no idea of what lies behind each door (it could be an object to clean, or an event that wastes the player’s time).

 
**4. When you think about this concept, what kind of “feel” are you getting? Is it a stressful / time based game? Or is it a chill / relaxing game where at the end you just feel satisfied with everything you cleaned up?**
If the player were to simply clean at their own pace, then the game could be a relaxing one (the nice purple hues of the prototype also help with this kind of chill atmosphere).

**Comment by LiliaIsABell** [comment]
**1. Is the cleaning up while procrastinating something relatable to you? If not, what other tasks do you do instead of what you’re supposed to do?**
I do find it relatable. Since it's something I have to do anyways, it feels justified to clean instead of working.

**2. Do you think the entire house would get too boring to clean?**
I don't think so if the tasks are different. For example, washing dishes in the kitchen, throwing away paper in the office, picking up laundry in the bedroom, etc.

**3. Do you have any suggestions for “surprise” events to pop up throughout the game? (maybe an online class that you need to run back to the bedroom to move your move / type in the chat, and you only have 30 seconds or you’ll get ‘caught’)?**
I think that the idea you mentioned is pretty good. The player can have a stamina bar and around the house there are snacks that energizes the player.

**4. When you think about this concept, what kind of “feel” are you getting? Is it a stressful / time based game? Or is it a chill / relaxing game where at the end you just feel satisfied with everything you cleaned up?**
Because of the previous question, I did see potential of it being a stressful and time based game. However, since it seems that your theme is procrastination, I think that it would work best if it was a more relaxing game that satisfies the brain with simply picking up stuff and seeing the room get cleaner.

**Comment by LeVieuxSinge** [comment]
**Is the cleaning up while procrastinating something relatable to you? If not, what other tasks do you do instead of what you’re supposed to do?**
It does feel like something I could relate to.

**Do you think the entire house would get too boring to clean?**
Cleaning the entire house would obviously feel satisfying to do as it is something I enjoy, but doing it constantly ? Maybe add some obstacles or enemies.

**Do you have any suggestions for “surprise” events to pop up throughout the game? (maybe an online class that you need to run back to the bedroom to move your move / type in the chat, and you only have 30 seconds or you’ll get ‘caught’)**
Your idea could work, but you should take a look at the game Roombo. It is very similar to what you want to do and plays around the idea of hide and seek.

**When you think about this concept, what kind of “feel” are you getting? Is it a stressful / time based game? Or is it a chill / relaxing game where at the end you just feel satisfied of everything you cleaned up?**
Stressful is not really how you would anticipate a room cleaning game so I'd say it is more chill. However, making it stressful could be different and could question the idea of being calm while cleaning our houses.
